### ISSUE TITLE
Quick Editor: Alt Text - Improve behaviour on the Alt text input text

### ISSUE THREAD

**Issue by hamorillo** [Automattic/Gravatar-SDK-Android#499]
The behavior of the placeholder can be improved. 

**Reminder:** We can't use experimental composables.

**Comment by AdamGrzybkowski** [comment]
@hamorillo I wonder if we need this. There's always a default alt text set, so the only way to show the placeholder is to delete the text manually. At that point, the placeholder doesn't provide much value.

**Comment by hamorillo** [comment]
Yeah, I agree. With the new default `altText`, the placeholder doesn't add much value. We can reduce the priority to `low` (just in case, at some point, we have time) or remove the ticket.
